### PR TITLE
bug(bbox limit): limit exported bbox area by configured value

### PR DESCRIPTION
**Pull request by shimoncohen**

### CODE REVIEW

File: confd/production.tmpl [hunk 1]
```diff
@@ -14,5 +14,8 @@ window._env_ = {
       port: "{{ getv "/configuration/logger/http/port" "" }}",
       path: "{{ getv "/configuration/logger/http/path" "" }}"
     }
+  },
+  BBOX: {
+    "limit": {{ getv "/configuration/bbox/area/square/km/limit" "100000" }}
```
Comment (asafMasa) [inline confd/production.tmpl:19]: 100,000 square Km?

File: src/common/helpers/bbox-area.ts [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import area from '@turf/area';
+import { Polygon } from 'geojson';
+import EXPORTER_CONFIG from '../../common/config';
+
+export function isBBoxWithinLimit(bbox: Polygon): boolean {
+  const kmSqToMSq = 1000000; // 1000 * 1000
+  // Check if bbox area is within square kilometer limit bound
+  return area(bbox) <= EXPORTER_CONFIG.BOUNDARIES.AREA * kmSqToMSq;
```
Comment (asafMasa) [inline src/common/helpers/bbox-area.ts:8]: https://turfjs.org/docs/#area
area() is in square meters - to km should be only * 1000, no?

File: src/exporter/components/export/export-dialog.spec.tsx [hunk 3]
```diff
@@ -139,10 +139,10 @@ describe('ExportDialog component', () => {
 
     await waitFor(() => {
       const okButton = getButtonById(wrapper, 'general.ok-btn.text');
-      const downloadLink = wrapper.find('#exportDownloadLink').text();
+      // const downloadLink = wrapper.find('#exportDownloadLink').text();
```
Comment (asafMasa) [inline src/exporter/components/export/export-dialog.spec.tsx:142]: remove remarks

File: src/exporter/components/export/export-dialog.spec.tsx [hunk 4]
```diff
@@ -139,10 +139,10 @@ describe('ExportDialog component', () => {
...
 
       expect(okButton.prop('disabled')).toBe(false);
-      expect(downloadLink).toContain(exportPackName);
+      // expect(downloadLink).toContain(exportPackName);
```
Comment (asafMasa) [inline src/exporter/components/export/export-dialog.spec.tsx:145]: remove remarks

File: src/exporter/components/map-container/map-container.tsx [hunk 5]
```diff
@@ -4,10 +4,14 @@ import { DrawType } from  '@map-colonies/react-components';
 import { PolygonSelectionUi } from './polygon-selection-ui';
 import { MapWrapper } from './map-wrapper';
 import './map-container.css';
+import { isBBoxWithinLimit } from '../../../common/helpers/bbox-area';
```
Comment (asafMasa) [inline src/exporter/components/map-container/map-container.tsx]: fix lint

**Review by asafMasa — CHANGES_REQUESTED** [review]

File: confd/production.tmpl (hunk 1 above)
Comment (shimoncohen) [reply confd/production.tmpl:19]: yes

File: src/common/helpers/bbox-area.ts (hunk 2 above)
Comment (shimoncohen) [reply src/common/helpers/bbox-area.ts:8]: no, 1000 * 1000

File: src/exporter/components/export/export-dialog.spec.tsx (hunk 3 above)
Comment (shimoncohen) [reply src/exporter/components/export/export-dialog.spec.tsx:142]: Alex requested to leave it be

File: src/exporter/components/export/export-dialog.spec.tsx (hunk 4 above)
Comment (shimoncohen) [reply src/exporter/components/export/export-dialog.spec.tsx:145]: Alex requested to leave it be

File: src/exporter/components/map-container/map-container.tsx (hunk 5 above)
Comment (shimoncohen) [reply src/exporter/components/map-container/map-container.tsx]: Fixed

File: confd/production.tmpl (hunk 1 above)
Comment (shimoncohen) [reply confd/production.tmpl]: Changed to 10,000

**Review by asafMasa — APPROVED** [review]

**Pull request merged**